convert2rhel/unit_tests/logger_test.py

```diff
@@ -73,9 +73,37 @@ def test_tools_opts_debug(tmpdir, read_std, is_py2):
 def test_logger_custom_logger(tmpdir, caplog):
     """Test CustomLogger."""
     log_fname = "convert2rhel.log"
-    logger_module.initialize_logger(log_name=log_fname, log_dir=str(tmpdir))
+    logger_module.setup_logger_handler(log_name=log_fname, log_dir=str(tmpdir))
     logger = logging.getLogger(__name__)
     logger.task("Some task")
     logger.file("Some task write to file")
     with pytest.raises(SystemExit):
         logger.critical("Critical error")
+
+
+@pytest.mark.parametrize(
+    ("log_name", "path_exists"),
+    (
+        ("convert2rhel.log", True),
+        ("convert2rhel.log", True),
+        ("convert2rhel.log", False),
+        ("convert2rhel.log", True),
+    ),
+)
```

Comment:
Since 3 of the runs are identical I'm guessing each test has side effects that affect the other runs. Ideally we'd have two parameterized tests here, one for path exists and one where it doesn't. And to test that multiple runs create archives I'd instead have a loop within the test itself

```suggestion
@pytest.mark.parametrize(
    ("log_name", "path_exists"),
    (
        ("convert2rhel.log", True),
        ("convert2rhel.log", False),
    ),
)
```